spec/db/migrate/20230419231330_add_user_fk_to_bookmarks_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
```

Comment:
Not sure if this test will be helpful long term, except maybe as an example?  Thoughts on if we should keep it in this PR?

Reply:
@sandbergja we can remove this test. Thanks.